src/common/extension-telemetry-events.ts

```diff
@@ -56,6 +56,7 @@ export const CONTENT_PAGE_OPENED: string = 'contentPageOpened';
 export const CONTENT_HYPERLINK_OPENED: string = 'contentHyperLinkOpened';
 export const CARD_SELECTION_TOGGLED: string = 'cardSelectionToggled';
 export const RULE_EXPANSION_TOGGLED: string = 'ruleExpansionToggled';
+export const CARDS_VISUAL_HELPER_TOGGLED: string = 'cardsVisualHelperToggled';
```

Comment:
Not used in this PR? (for future telemetry?)

Reply:
Good catch: yes, not used here. I'll remove it and add it in a future PR.

Unfortunately it seems nothing is firing telemetry on the Cards UI due to some missing items in the message/action creator side of things, so I'll cover that in another PR.